packages/client/hmi-client/src/components/workflow/ops/calibrate-ciemss/calibrate-utils.ts

```diff
@@ -58,22 +59,24 @@ export function getErrorData(groundTruth: DataArray, simulationData: DataArray,
 	);
 	if (relevantGroundTruthColumns.length === 0) return errors;
 
+	// Fix the time values by using the index of the value since the values are floats not integer e.g t: 0.904522613065327 instead of t: 1
+	// We assume that ground truth is sorted by time in ascending order
+	const groundTruthTimeValuesCorrected = groundTruth.map((ele, index) => ({ ...ele, [datasetTimeCol]: index }));
```

Comment:
You can't make this assumption of using the index - what is the problem here?

Reply:
So then this may be something that @Tom-Szendrey needs to resolve in the MAE function, since we are already doing matching resolutions on time-point there. Maybe check with @liunelson to see if this is an immediate concern that needs to be addressed or we can back-burn it.

The data you are showing looks like it came from Julia's simulation, which prioritizes on emulating the accurate shape of the curve rather than providing useful forecast output.

In the meantime @jryu01 use a different dataset for ground truth:
- Run a simulation
- Grab the dataset of the simulation from S3 and use that as the ground truth

Reply:
Hm yeah.. it looks like the dataset I was testing with have such weirdness for `t`, I'll grab different dataset and test with it  again.